### PR TITLE
Add some pt-br translations on code(colors, commands, etc)

### PR DESCRIPTION
**Pull request by PHGaming121**

### CODE REVIEW

File: Patches/ChatCommandPatch.cs [hunk 1]
```diff
@@ -1198,27 +1232,32 @@ public static void OnReceiveChat(PlayerControl player, string text, out bool can
...
-                        Utils.ShowAllActiveSettings(player.PlayerId);
-                        break;
-                    default:
-                        Utils.ShowActiveSettings(player.PlayerId);
-                        break;
+                        case "r":
+                        case "roles":
+                        case "funções":
+                            Utils.ShowActiveRoles();
+                            break;
```
Comment (Tommy-XL) [inline Patches/ChatCommandPatch.cs:1252]: Why `player.PlayerId` was removed in `Utils.ShowActiveRoles()`

**Comment by Tommy-XL** [comment]
And need fix conflicts

**Comment by PHGaming121** [comment]
oh wrong one

**Review by Tommy-XL — APPROVED** [review]

**Comment by Tommy-XL** [comment]
Damn, again need fix conflicts

**Pull request merged**